### DISCUSSION TITLE
headlessui transition is not working

### DISCUSSION THREAD

**Discussion by AdlanHawari** [tailwindlabs/headlessui#1350] [Help]
I usually can create a dropdown using headlessui dropdown in nextjs project, but yesterday I create a new project with the following dependency

```
"dependencies": {
    "@headlessui/react": "^1.5.0",
    "next": "12.1.5",
},

  "devDependencies": {
    "autoprefixer": "^10.4.4",
    "eslint": "8.13.0",
    "eslint-config-next": "12.1.5",
    "postcss": "^8.4.12",
    "tailwindcss": "^3.0.24"
  }
```
Dropdown works when i remove the Transition component. But I need to customize the transition (open the dropdown when hovering). Is there any solution for this problem?

**Comment by RobinMalfait** [comment]
Hey! Thank you for your question!
Much appreciated! 🙏

Please checkout #681 for more information.
